include/onnxruntime/core/graph/graph.h

```diff
@@ -1760,6 +1760,15 @@ class Graph {  // NOLINT(clang-analyzer-optin.performance.Padding): preserve exi
                                                     std::vector<const ONNX_NAMESPACE::TypeProto*>& output_types,
                                                     const Graph::ResolveOptions& options);
 
+  // If the shape values are inferred after executing ONNX operator's PartialDataPropagationFunction(),
```

Comment:
It is not always shape of the output, but the shape is in the output. I think the wording may need to be updated to make it clear.